maintenance/wikia/compareMessagingWiki.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+require_once( __DIR__ . '/../Maintenance.php' );
+
+class CompareMessagingWiki extends Maintenance {
+
+	const MESSAGING_WIKI_DB = 'messaging';
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Compares i18n messages in the code and customized on messaging.wikia.com";
+
+		$this->addOption( 'messages', 'Pipe separated list of messages to check', true, true );
+	}
+
+	/**
+	 * @return array
+	 */
+	private function getMessagesFiles() {
+		global $wgExtensionMessagesFiles;
+
+		return array_values(array_unique(
+			array_merge(
+				GlobalMessagesService::getInstance()->getCoreMessageFiles(),
+				GlobalMessagesService::getInstance()->getExtensionMessageFiles(),
+				$wgExtensionMessagesFiles
+			)
+		));
+	}
+
+	public function getMessageFromCode($msg) {
+		$files = $this->getMessagesFiles();
+		$ret = [];
+
+		foreach($files as $file) {
+			include $file;
+
+			foreach($messages as $lang => $langMessages) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$messages" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)